tasks.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+"""
+Task execution tool & library
+"""
+
+import json
+import sys
+from logging import basicConfig, getLogger
+from pathlib import Path
+
+import docker
+import git
+from bumpversion.cli import main as bumpversion
+from invoke import task
+from veracode.__init__ import __version__  # TODO: Get globally available version
```

Comment:
TODO - are we able to pull this from setup.cfg? It pulls the version the same way though.

Reply:
You can, or you can have bumpversion automatically update a hard coded variable in `tasks.py`